packages/dds/merge-tree/src/test/client.rollback.spec.ts

```diff
@@ -399,4 +399,96 @@ describe("client.rollback", () => {
 
 		assert.equal(segment.parent, undefined);
 	});
+	// similar to an existing test - is it worth checking that we can rollback to empty?
```

Comment:
For both this test and the last test I added - the cases we already have are somewhat similar, but I wasn't sure if they covered different enough cases to add value. If they don't add anything, I can just remove them.

Reply:
Which other cases do you think they're similar to?

Reply:
"Should rollback delete across split segments" and "Should rollback annotates with multiple previous property sets," respectively, but I'm now thinking that the multiple annotates cases are different enough due to the remove operation in the case I've added.